tdrs-frontend/src/App.test.js

```diff
@@ -2,12 +2,68 @@ import React from 'react'
 import { shallow } from 'enzyme'
 
 import GovBanner from './components/GovBanner'
+import Header from './components/Header'
+import { Alert } from './components/Alert'
 
 import App from './App'
 
 describe('App.js', () => {
+  afterEach(() => {
```

Comment:
This reset the window location so that the tests pass even when the value is changed inside the test.